src/libCli/GrammarConstruction.cpp

```diff
@@ -23,31 +23,30 @@ using namespace ArgParse;
 namespace cli
 {
 
-std::string getServerUri(ParsedElement * f_parseTree)
-{
-    std::string serverUri = f_parseTree->findFirstChild("ServerUri");
-    if(f_parseTree->findFirstChild("TcpUri") != "" and f_parseTree->findFirstChild("TcpPort") == "")
+    std::string getServerUri(ParsedElement *f_parseTree)
     {
-        serverUri += ":50051";
+        std::string serverUri = f_parseTree->findFirstChild("ServerUri");
+        if (f_parseTree->findFirstChild("TcpUri") != "" and f_parseTree->findFirstChild("TcpPort") == "")
+        {
+            serverUri += ":50051";
+        }
+        //serverUri = "127.0.0.1:50051";
+        return serverUri;
     }
-    //serverUri = "127.0.0.1:50051";
-    return serverUri;
-}
 
-class GrammarInjectorMethodArgs : public GrammarInjector
-{
+    class GrammarInjectorMethodArgs : public GrammarInjector
+    {
     public:
-        GrammarInjectorMethodArgs(Grammar & f_grammar, const std::string & f_elementName = "") :
-            GrammarInjector("MethodArgs", f_elementName),
-            m_grammar(f_grammar)
+        GrammarInjectorMethodArgs(Grammar &f_grammar, const std::string &f_elementName = "") : GrammarInjector("MethodArgs", f_elementName),
```

Comment:
fix formatting: after `:` newline and every initializer in ner line with correct indentation then.